.travis.yml

```diff
@@ -17,6 +17,7 @@ before_install:
   - sudo hostname "$(hostname | cut -c1-63)"
   - sed -e "s/^\\(127\\.0\\.0\\.1.*\\)/\\1 $(hostname | cut -c1-63)/" /etc/hosts | sudo tee /etc/hosts
   - cat /etc/hosts # optionally check the content *after*
+  - yes | sdkmanager "platforms;android-26"
```

Comment:
did you mean this to be `28`?

Reply:
I tried that initially but that didn't work so I switched back to the 26. I found this post indicating that there is some bug related to the acceptance of the 28 licence. https://travis-ci.community/t/accept-license-for-android-build-tools-27-0-3-bug/549

Reply:
I think you can do this `printf "d56f5187479451eabf01fb78af6dfcb131a6481e\n24333f8a63b6825ea9c5514f83c2829b004d1fee" > android-sdk-linux/licenses/android-sdk-license` as a work around